prospero-metadata/src/main/java/org/wildfly/prospero/metadata/ProsperoMetadataUtils.java

```diff
@@ -53,6 +53,8 @@ public class ProsperoMetadataUtils {
 
     private static final String WARNING_MESSAGE = "WARNING: The files in .installation directory should be only edited by the provisioning tool.";
 
+    public static final String CANDIDATE = ".candidate.txt";
```

Comment:
Could you move the constant to PrepareCandidateAction. prospero-metadata should contain only information needed by external tools to work with prospero, I don't think candidate information is needed.

Also could you rename this to `candidate_artifact_channels.txt` or something similar?

Reply:
> Could you move the constant to PrepareCandidateAction. prospero-metadata should contain only information needed by external tools to work with prospero, I don't think candidate information is needed.
> 
> Also could you rename this to `candidate_artifact_channels.txt` or something similar?

Hi @spyrkob,
This constant is also used in ApplyCandidateAction so it has to be kept in common place.